opm/io/eclipse/ERst.hpp

```diff
@@ -51,6 +51,7 @@ class ERst : public EclFile
     }
 
     int count(const std::string& name, int reportStepNumber) const; 
+    int numberOfReportSteps() const { return seqnum.size(); };
```

Comment:
seqnum is  a vector, it's size is size_t, not int. i do not see see why we would want make it signed and truncated.

Reply:
> it's size is size_t,

It actually isn't.  The return value from `vector<T>::size()` is really
```C++
std::vector<T>::size_type
```
which **happens to be** `std::size_t` in many implementations.

Reply:
done